docs/screens/login_screen.md

```diff
@@ -0,0 +1,29 @@
+# Login Screen
+
+<img src="../analytics_imgs/login/login_screen.png" width=20% height=20%>
+
+## Documentation
+
+|                         When user clicks log in.                          | Specification                                                                    |
+|:-------------------------------------------------------------------------:|----------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/login/login_screen.png" width=50% height=50%> | <pre>{<br />   "name": "button_click"<br />   "item_name": "log_in"<br />}<pre/> |
+
+|                         When github auth opens.                          | Specification                                                                               |
+|:------------------------------------------------------------------------:|---------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/login/github_auth.png" width=50% height=50%> | <pre>{<br />   "name": "simple_action"<br />   "item_name": "open_github_auth"<br />}<pre/> |
+
+|                       When user clicks log in on xiaomi device.                       | Specification                                                                                     |
```

Comment:
**chore**: when user click, we should fire "button_click" event, if we open a screen, we should fire `screen_opened`

**chore**: it is much easier if only one screen is presented on single page

Reply:
What should be here in your opinion? After click log in on xiaomi device, the permission dialog is displayed.